features/bevragen/persoon/partner/aangaan-huwelijk-partnerschap/dev/aangaanHuwelijkPartnerschap-gba.feature

```diff
@@ -0,0 +1,44 @@
+# language: nl
+
+Functionaliteit: aangaan huwelijk/partnerschap bij actuele of ontbonden relatie
+
+  Rule: aangaanHuwelijkPartnerschap wordt -indien gevraagd met fields- geleverd voor zowel een actuele als een ontbonden huwelijk of partnerschap
+
+    Scenario: partnerschap is actueel
+      Gegeven de persoon met burgerservicenummer '000000012' heeft een 'partner' met de volgende gegevens
+      | naam                                                                | waarde    |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10)  | 20010809  |
+      | plaats huwelijkssluiting/aangaan geregistreerd partnerschap (06.20) | Parijs    |
+      | land huwelijkssluiting/aangaan geregistreerd partnerschap (06.30)   | 5002      |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                               |
+      | type                | RaadpleegMetBurgerservicenummer      |
+      | burgerservicenummer | 000000012                            |
+      | fields              | partners.aangaanHuwelijkPartnerschap |
+      Dan heeft de response een persoon met een 'partner' met alleen de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam                | waarde    |
+      | datum               | 20010809  |
+      | plaats.omschrijving | Parijs    |
+      | land.code           | 5002      |
+      | land.omschrijving   | Frankrijk |
+
+    Scenario: partnerschap is ontbonden
+      Gegeven de persoon met burgerservicenummer '000000012' heeft een 'partner' met de volgende gegevens
```

Comment:
huwelijksluiting/aangaan gps gegevens moet dus uit de rij met volg_nr 1 worden gehaald. In de test database kom ik ook een persoon tegen waar zowel huwelijksluiting/aangaan gps en ontbinding gegevens in de rij met volg_nr 0 staan (pl_id=1554). Is dit een fout of kan dit ook voorkomen in productie? Als het wel voorkomt in productie, dan neem ik aan dat huwelijksluiting gegevens uit de rij met volg_nr 0 moet worden gehaald?

Reply:
in het LO staat "De groepen 06 en 07 komen niet tegelijk voor." Dus aangaan/sluiting (groep 06) en ontbinding (groep 07) kunnen niet allebei in volg_nr 0 zitten. Als dit voorkomt is het een datafout.